packages/webapp/src/components/CreateUserAccount/index.jsx

```diff
@@ -22,31 +23,69 @@ export default function PureCreateUserAccount({ onSignUp, email, onGoBack }) {
   } = useForm({
     mode: 'onTouched',
   });
+
   const NAME = 'name';
   const GENDER = 'gender';
+  const LANGUAGE = 'language';
   const BIRTHYEAR = 'birth_year';
   const PASSWORD = 'password';
   const password = watch(PASSWORD, undefined);
   const { t } = useTranslation(['translation', 'common', 'gender']);
   const title = t('CREATE_USER.TITLE');
+
   const {
     isValid: isPasswordValid,
     hasNoSymbol,
     hasNoDigit,
     hasNoUpperCase,
     isTooShort,
   } = validatePasswordWithErrors(password);
+
   const genderOptions = [
     { value: 'MALE', label: t('gender:MALE') },
     { value: 'FEMALE', label: t('gender:FEMALE') },
     { value: 'OTHER', label: t('gender:OTHER') },
     { value: 'PREFER_NOT_TO_SAY', label: t('gender:PREFER_NOT_TO_SAY') },
   ];
 
+  const languageOptions = [
+    { value: 'en', label: t('PROFILE.ACCOUNT.ENGLISH') },
+    { value: 'es', label: t('PROFILE.ACCOUNT.SPANISH') },
+    { value: 'pt', label: t('PROFILE.ACCOUNT.PORTUGUESE') },
+    { value: 'fr', label: t('PROFILE.ACCOUNT.FRENCH') },
+  ];
+
+  const getLanguageOption = (language) => {
+    switch (language) {
+      case 'en':
+        return 0;
+      case 'es':
+        return 1;
+      case 'pt':
+        return 2;
+      case 'fr':
+        return 3;
+    }
+  };
+
+  const browser_langauge = navigator.language.includes('-')
+    ? navigator.language.split('-')[0]
+    : navigator.language;
+
+  const [language, setLanguage] = useState(browser_langauge);
+  const [languageOption, setLanguageOption] = useState(getLanguageOption(language));
+
+  useEffect(() => {
+    setLanguageOption(getLanguageOption(language));
+    i18n.changeLanguage(language);
+    localStorage.setItem('litefarm_lang', language);
```

Comment:
can put 'litefarm_lang' in the constants 
e.g 

export const LITEFARM_LANG = 'litefarm_lang'

Reply:
That's a good idea. But litefarm_lang is used throughout the app (in containers and components of webapp) and we do not have a constants file at the head of the directory. Should we create it?